foundations/javascript_basics/developer_tools_2.md

```diff
@@ -13,6 +13,8 @@ There are three ways to open the Developer Tools menu:
 
 ### Assignment
 
+Note: Google has updated the linked page since this lesson's creation, but students should still utilize the resource accordingly in regard to the lesson instructions.
```

Comment:
I think this should be re-worded to "Note: Google has updated the linked pages below, so some things may be missing if you are trying to follow along using the dev tools."

And I think this note should be moved to  line 21!